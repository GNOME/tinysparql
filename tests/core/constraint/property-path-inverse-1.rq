CONSTRAINT GRAPH <http://example.com/A>
SELECT ?s ?o {
  ?s !(nrl:modified|nrl:added|rdfs:subClassOf) ?o .
  FILTER (?o != rdfs:Class && ?o != rdfs:Resource) .
}
ORDER BY ?s ?o
