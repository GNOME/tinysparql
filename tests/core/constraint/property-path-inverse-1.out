"http://example/a"	"http://example/A"
